lib/src/common/variables.ts

```diff
@@ -1,709 +1,541 @@
   },
   dialog: {
-    overlayColor: globalTokens.color_grey_a_800,
-    backgroundColor: globalTokens.hal_white,
+    overlayColor: CORE_TOKENS.color_grey_800_a,
+    backgroundColor: CORE_TOKENS.color_white,
     closeIconWidth: "24px",
     closeIconHeight: "24px",
     closeIconTopPosition: "20px",
     closeIconRightPosition: "20px",
-    closeIconBackgroundColor: globalTokens.transparent,
-    closeIconBorderColor: globalTokens.border_none,
-    closeIconColor: globalTokens.hal_black,
-    closeIconBorderThickness: globalTokens.border_width_0,
-    closeIconBorderStyle: globalTokens.border_solid,
+    closeIconBackgroundColor: CORE_TOKENS.color_transparent,
+    closeIconBorderColor: CORE_TOKENS.border_none,
+    closeIconColor: CORE_TOKENS.color_black,
+    closeIconBorderThickness: CORE_TOKENS.border_width_0,
+    closeIconBorderStyle: CORE_TOKENS.border_solid,
     closeIconBorderRadius: "2px",
     boxShadowOffsetX: "0px",
     boxShadowOffsetY: "1px",
     boxShadowBlur: "3px",
-    boxShadowColor: globalTokens.color_grey_a_300,
+    boxShadowColor: CORE_TOKENS.color_grey_300_a,
   },
   dropdown: {
-    buttonBackgroundColor: globalTokens.hal_white,
-    hoverButtonBackgroundColor: globalTokens.hal_grey_l_95,
-    activeButtonBackgroundColor: globalTokens.lightGrey,
-    buttonFontFamily: globalTokens.type_sans,
-    buttonFontSize: globalTokens.type_scale_03,
-    buttonFontStyle: globalTokens.type_normal,
-    buttonFontWeight: globalTokens.type_regular,
-    buttonFontColor: globalTokens.hal_black,
+    buttonBackgroundColor: CORE_TOKENS.color_white,
+    hoverButtonBackgroundColor: CORE_TOKENS.color_grey_100,
+    activeButtonBackgroundColor: CORE_TOKENS.color_grey_300,
+    buttonFontFamily: CORE_TOKENS.type_sans,
+    buttonFontSize: CORE_TOKENS.type_scale_03,
+    buttonFontStyle: CORE_TOKENS.type_normal,
+    buttonFontWeight: CORE_TOKENS.type_regular,
+    buttonFontColor: CORE_TOKENS.color_black,
     buttonIconSize: "20px",
     buttonIconSpacing: "10px",
-    buttonIconColor: globalTokens.hal_black,
+    buttonIconColor: CORE_TOKENS.color_black,
     buttonPaddingTop: "0px",
     buttonPaddingBottom: "0px",
     buttonPaddingLeft: "16px",
     buttonPaddingRight: "16px",
-    disabledColor: globalTokens.hal_grey_l_60,
-    disabledButtonBackgroundColor: globalTokens.transparent,
-    disabledBorderColor: globalTokens.lighterBlack,
-    optionBackgroundColor: globalTokens.hal_white,
-    hoverOptionBackgroundColor: globalTokens.hal_grey_l_95,
-    activeOptionBackgroundColor: globalTokens.hal_grey_l_80,
-    optionFontFamily: globalTokens.type_sans,
-    optionFontSize: globalTokens.type_scale_root,
-    optionFontStyle: globalTokens.type_normal,
-    optionFontWeight: globalTokens.type_regular,
-    optionFontColor: globalTokens.hal_black,
+    disabledColor: CORE_TOKENS.color_grey_500,
+    disabledButtonBackgroundColor: CORE_TOKENS.color_transparent,
+    disabledBorderColor: CORE_TOKENS.color_transparent,
```

Comment:
This is a special case. `disabledBorderColor` doesn't apply any style to the regular component, it is a customizable token to add a border to the component when the state is disabled. So, by default, It doesn't matter its color since the border width is 0px and doesn't make any difference.

Reply:
This is a special case. `disabledBorderColor` doesn't apply any style to the regular component, it is a customizable token to add a border to the component when the state is disabled. So, by default, It doesn't matter its color since the border width is 0px and doesn't make any difference.